pwnlib/data/templates/pwnup.mako

```diff
@@ -131,13 +131,12 @@ def start(argv=[], *a, **kw):
 %endif
 gdbscript = '''
 %if ctx.binary:
+set sysroot
   %if 'main' in ctx.binary.symbols:
-break *0x{exe.symbols.main:x}
-  %else:
-break *0x{exe.entry:x}
+break main
```

Comment:
This is probably a good plan since `exe.symbols.main` will be incorrect for PIE / ASLR-enabled binaries.

Reply:
We should also probably change this to `tbreak`, so that the breakpoint is removed after `main` is hit.

Reply:
Changed to `tbreak`.